### PR TITLE
Build: Remove unused Linear dependencies

### PR DESCRIPTION
**Pull request by ndelangen**

### CODE REVIEW

File: code/frameworks/react-vite/package.json [hunk 1]
```diff
@@ -48,7 +48,7 @@
   },
   "dependencies": {
     "@joshwooding/vite-plugin-react-docgen-typescript": "0.2.1",
-    "@rollup/pluginutils": "^4.2.0",
```
Comment (yannbf) [inline code/frameworks/react-vite/package.json:51]: This doesn't seem to be related to linear

Comment (ndelangen) [reply code/frameworks/react-vite/package.json:51]: I accidentally merged PRs in the wrong order:
https://github.com/storybookjs/storybook/pull/22960
https://github.com/storybookjs/storybook/pull/22961

... got merged in to this, because I telescoped a bunch of times to not waste time on merge conflicts.

This commit is the only commit of note on this PR:
[`92d4915` (#22958)](https://github.com/storybookjs/storybook/pull/22958/commits/92d49157a642951c3f077dc27c4aba0ccee3b006)
